pyanaconda/modules/subscription/subscription.py

```diff
@@ -330,13 +328,6 @@ def set_system_purpose_with_task(self):
 
     # subscription request
 
-    def _validate_subscription_request_type(self, request_type):
```

Comment:
All these look like aritifacts of the Fedora <-> RHEL rebase passes & should be fine to drop.